### PR TITLE
Validate resolved federation address

### PR DESCRIPTION
**Pull request by quietbits**

### CODE REVIEW

File: src/constants/settings.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import StellarSdk from "stellar-sdk";
 
-// TODO: set to 100 before launch
-export const TX_HISTORY_LIMIT = 20;
+export const TX_HISTORY_LIMIT = 100;
```
Comment (leighmcculloch) [inline src/constants/settings.ts:3]: ❓ Is this related to resolving a federation address, or accidental change? Looks unrelated to me.

**Review by leighmcculloch — APPROVED** [review]
One question, regardless 🎉 .

File: src/constants/settings.ts (hunk 1 above)
Comment (quietbits) [reply src/constants/settings.ts:3]: @leighmcculloch It's unrelated to resolving the federation address. I just snuck it in there because it was so tiny. :blush:

File: src/components/SendTransaction/CreateTransaction.tsx [hunk 2]
```diff
@@ -212,50 +213,55 @@ export const CreateTransaction = ({
...
     } else {
       resetFederationAddressInput();
     }
   };
 
-  const checkIfAccountIsFunded = async () => {
-    setIsCheckingAddress(true);
-
-    if (!toAccountId || !StrKey.isValidEd25519PublicKey(toAccountId)) {
+  const checkIfAccountIsFunded = async (accountId: string) => {
```
Comment (acharb) [inline src/components/SendTransaction/CreateTransaction.tsx:254]:
```suggestion
  const checkAndSetIsAccountFunded = async (accountId: string) => {
```

Comment (acharb) [reply src/components/SendTransaction/CreateTransaction.tsx:254]: Another option could be `setIsAccountFunded(!accountId || !StrKey.isValidEd25519PublicKey(accountId)`, the wrapper method name is kind of ambiguous what it's doing imo (it checks, and then what)

**Review by acharb — APPROVED** [review]
LGTM, 1 naming nit but up to you to make change or not

File: src/components/SendTransaction/CreateTransaction.tsx (hunk 2 above)
Comment (quietbits) [reply src/components/SendTransaction/CreateTransaction.tsx:254]: Good point, `checkAndSetIsAccountFunded` is more clear. `setIsAccountFunded` is already taken for updating the state value.
